packages/arb-tx-aggregator/txdb/txdb.go

```diff
@@ -404,6 +491,22 @@ func (db *TxDB) saveAssertion(ctx context.Context, processed processedAssertion)
 			return err
 		}
 
+		ethLogsList, err := db.GetLogs(ctx, block.Hash())
+		if err != nil {
+			return err
+		}
```

Comment:
Rather than doing this, I think it'd be better to get the logs from `ethReceipts` defined above